rosetta-source/src/main/rosetta/base-staticdata-asset-common-enum.rosetta

```diff
@@ -13,6 +13,8 @@ enum AssetClassEnum: <"The enumerated values to specify the FpML asset class cat
 
 enum AssetIdTypeEnum extends ProductIdTypeEnum: <"Extends product identifiers with additional identifier sources for Assets.">
     CurrencyCode <"Used to identify the currency of a Cash Asset.">
+    ExchangeCode <"The identifier follows the symbology set by the exchange which lists the asset.">
+    ClearingCode <"The identifier follows the symbology set by the clearing house which clears the asset.">
```

Comment:
I thought this was going in Phase 2?

Reply:
It was a non-impactful change so brought forward